services/datamanager/builtin/disk_summary_logger_windows.go

```diff
@@ -0,0 +1,5 @@
+package builtin
```

Comment:
does this need a `go:build` flag to ensure it only builds on windows?

Reply:
go will treat a _GOOS.go file ending as a build constraint ([docs](https://pkg.go.dev/cmd/go#hdr-Build_constraints))